nachos/src/main/java/com/hootsuite/nachos/NachoTextView.java

```diff
@@ -1127,4 +1139,8 @@ public boolean onSingleTapUp(MotionEvent e) {
             return true;
         }
     }
+
+    public interface OnChipAddListener {
+        void onChipAdded(Chip chip);
```

Comment:
Can you provide the documentation and contract of when and what the `onChipAdded` is supposed to adhere to?  It can be similar to the `onChipRemove`